example/config/app.conf.json

```diff
@@ -98,11 +98,82 @@
           "membership": {"subsystem": "metering"},
           "appearance": {"title": "PV Array"},
           "location": {"title": "Roof", "floor": "Roof"}
+        },
+        {
+          "name": "smart-core/meters/03",
+          "traits": [{"name": "smartcore.bos.Meter", "more": {"unit":  "litres"}}],
```

Comment:
Interesting approach to this. I expect we'll need something else for more complicated mock setup, but it works 👍